### ISSUE TITLE
 where should i put pre-trained neural network (VGG19, 80Mb)?

### ISSUE THREAD

**Issue by jacktive** [alexjc/neural-enhance#119]
i unzip the neural-enhance-master.zip to a folder,and put all the pre-trained neural network (VGG19, 80Mb) in it. is that right?
i run that "python3 enhance.py --type=photo --model=deblur --zoom=1 pre.jpg" without the pre-trained neural network (VGG19, 80Mb) file,it did not hint any erro or waring.
what should i do to test Example #3 ?

**Comment by ikarth** [comment]
Yes, the pre-trained VGG19 network file goes in the same folder as enhance.py.

**Comment by pantlmn** [comment]
`python3 enhance.py --type=photo --model=deblur --zoom=1 pre.jpg` will use pre-trained network file `ne1x-photo-deblur-0.3.pkl.bz2`, not the `vgg19_conv.pkl.bz2`.

I also wish to know how to reproduce example #3. But AFAIK, the file of the network, trained on the faces of celebrities is not yet available here.

**Comment by JanGross** [comment]
What exactly has vgg19_conv.pkl.bz2 been trained on? Couldn't really figure this out

**Comment by milinddeore** [comment]
@alexjc Can you please kindly update on face model and also what vgg19_conv.pkl.bz2 model being trained on? Thanks so much!

**Comment by Jurph** [comment]
VGG19 is a pre-trained model (see its predecessors VGG16 et al). It's trained on ImageNet, which is a pretty enormous dataset. Alex's model piggy-backs on VGG19, using one of the VGG19 layers as the "loss network", effectively leveraging thousands of hours of training and work so that the model can measure how well it's doing.

The model trained on celebrity faces probably isn't available, but I bet the dataset is CELEB-A or CELEBA-HQ. Training on that dataset could take a day or two but then you'll have your own model.

**Comment by theultraman20** [comment]
Is the code ever going to be updated to allow us to enhance faces using the vgg19_conv.pkl.bz2 model? Is there a way to use the model that I don't know about?
